### DISCUSSION TITLE
music does not play - only silence is played

### DISCUSSION THREAD

**Discussion by peqoud** [mikebrady/shairport-sync#1631] [Q&A] [answered]
I have installed the latest version of shairport-sync an NQPTP

- shairport-sync: 4.1-dev-875-gf164f19a-AirPlay2-alac-OpenSSL-Avahi-ALSA-pa-soxr-sysconfdir:/etc
- NQPTP: Version: 1.1-dev-200-g5762735. Shared Memory Interface Version: 9.

Connecting works fine and I see the device. Though music never starts and it looks like only silence is played.
Checking the log files I see the following:

```
Line  1841:          0.005899544 "player.c:1266" Sent 8820 frames of silence
Line  1849:          0.000246913 "player.c:1266" Sent 199 frames of silence
Line  1857:          0.000178740 "player.c:1266" Sent 366 frames of silence
Line  1865:          0.000140387 "player.c:1266" Sent 43 frames of silence
```

This seems to stop with:

`0.000028105 "player.c:2754" first frame sync error (positive --> late): 6 frames, 0.136 mS at 44100 frames per second output.`

Logfile attached: [shairport-sync-log.txt](https://github.com/mikebrady/shairport-sync/files/10785978/shairport-sync-log.txt)

Not sure what else to check to get this right.

**Comment by mikebrady** [comment]
Thanks. I think this should really be an issue rather than a discussion -- would be be kind enough to open an issue about it?

**Comment by peqoud** [comment, marked as answer]
Issue created #1638
